tests/components/airzone/test_binary_sensor.py

```diff
@@ -1,11 +1,12 @@
 """The sensor tests for the Airzone platform."""
 
 from homeassistant.const import STATE_OFF, STATE_ON
+from homeassistant.core import HomeAssistant
 
 from .util import async_init_integration
 
 
-async def test_airzone_create_binary_sensors(hass):
+async def test_airzone_create_binary_sensors(hass: HomeAssistant):
```

Comment:
Typing is incomplete: Missing return type.

Reply:
Fixed in 384bb573ae153c25ff24e1ba86649b0739d9349e